presto-cli/pom.xml

```diff
@@ -170,6 +183,96 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>rpm-maven-plugin</artifactId>
+                <version>2.2.0</version>
+                <executions>
+                    <execution>
+                        <id>generate-rpm</id>
+                        <goals>
+                            <goal>rpm</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <name>${project.name}</name>
+                    <version>${project.version}</version>
+                    <release>1</release>
+                    <group>Applications/Databases</group>
+                    <description>Presto CLI RPM Package.</description>
+                    <needarch>x86_64</needarch>
+                    <copyTo>${project.build.directory}/rpm/${project.build.finalName}.x86_64.rpm</copyTo>
```

Comment:
I added this to copy the RPM to a more easier path that can referenced later by enforcer and integration tests.
The original path looks like `presto-cli/target/rpm/presto-cli-rpm/RPMS/x86_64/presto-cli-rpm-335-1.x86_64.rpm` ie. `${project.build.directory}/rpm/${project.name}-rpm/RPMS/x86_64/${project.name}-rpm-${project.truncatedVersion}-1.x86_64.rpm` which is not easy to template.
Unfortunately the maven-rpm-plugin doesn't inject any properties to the build to make this easier. The only other option was to copy the final rpm to a more friendly path which is what I've done for now.